api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java

```diff
@@ -530,262 +529,12 @@ public void testGetRuntime_overrideLabel() throws ApiException {
   public void testGetRuntime_noGetRuntime_emptyListRuntimes() throws ApiException {
     when(mockUserRuntimesApi.getRuntime(GOOGLE_PROJECT_ID, getRuntimeName()))
         .thenThrow(new ApiException(404, "Not found"));
-    when(mockUserRuntimesApi.listRuntimesByProject(GOOGLE_PROJECT_ID, null, true))
+    when(mockUserRuntimesApi.listRuntimesByProject(GOOGLE_PROJECT_ID, null))
         .thenReturn(Collections.emptyList());
 
     assertThrows(NotFoundException.class, () -> runtimeController.getRuntime(WORKSPACE_NS));
   }
 
-  @Test
```

Comment:
The tests below were all verifying that a value was retrieved from Leo's ListRuntime endpoint. [We are no longer using that for GetRuntime](https://github.com/all-of-us/workbench/pull/9209/files#r2038140787), so these tests are no longer relevant.

Reply:
great - and the previous test covers the remaining behavior